### ISSUE TITLE
Need maintainer to activate Github workflows

### ISSUE THREAD

**Issue by actual-nh** [CleverRaven/Cataclysm-DDA#48680]
# Describe the bug

Github isn't doing Actions/Workflows for new contributors; see #48678 for an example. Unfortunately, they've chosen to be uncooperative and require maintainer privileges to activate them - anyone with permission to close other people's PRs, such as triage, should be able to do this. (This is pretty obviously them trying to reduce how many Actions run - understandable in the case of inactive projects that people are likely using to run other things, but by making it maintainers it's pretty obvious that they don't care about new contributors to active projects.)

# Expected behavior

I suggest doing 2 things (as well as activating workflows on the PR mentioned above, of course):
- Adding a label for "needs maintainer to run workflows" that the triage team can put on PRs.
- Someone who's paying Github (Pro or whatever) complain that it shouldn't require a maintainer to activate workflows for new contributors. (From my prior experience, they won't listen to anyone not paying them unless you hand them a no-cost-to-them solution on a silver platter.)

**Comment by BrettDong** [comment]
The narrative from GitHub is that some accounts are mining crypto currency through GitHub Action workflows by opening pull requests on other people's public repositories, and this is a counter measure they've come up with.

**Comment by ZhilkinSerg** [comment]
This is a security measure - GH is preventing running various miners on their infrastructure.

**Comment by actual-nh** [comment]
Yes, that's what I thought. That they're heavily restricting who can run the workflows indicates that they're also probably somewhat overloaded by legitimate stuff, so are trying to cut wherever they can find an excuse.

**Comment by Maleclypse** [comment]
Until we come up with a better solution feel free to ping me if you notice something needs to run.  I'll get look at it as quickly as I can.

**Comment by Night-Pryanik** [comment]
Closing as stale, since stalebot can't do this by itself.
